test/framework/universal_cluster.go

```diff
@@ -395,6 +395,10 @@ func (c *UniversalCluster) DeleteMesh(mesh string) error {
 	return c.GetKumactlOptions().KumactlDelete("mesh", mesh, "")
 }
 
+func (c *UniversalCluster) DeletePolicy(kumaType string, name string, meshName string) error {
```

Comment:
IMHO it's not worth introducing this function. Cluster object is already very polluted :/

Reply:
you are right, removed